README.md

```diff
@@ -96,27 +96,22 @@ Optionally, change the `fail-build` field to `false` to avoid failing the build
+| `image`             | The image to scan, this is mutually exclusive to `path`                                                                                                                                                                                                                                                        | N/A           |
+| `path`              | The file path to scan, this is mutually exclusive to `image`                                                                                                                                                                                                                                                   | N/A           |
 | `debug`             | Verbose logging output                                                                                                                                                                                                                                                                                         | `false`       |
-| `fail-build`        | Fail the build if a vulnerability is found with a higher severity. That severity defaults to `"medium"` and can be set with `severity-cutoff`.                                                                                                                                                                 | `false`       |
-| `grype-version`     | An optional parameter to specify a specific version of `grype` to use for the scan. Default is the version locked to the scan-action release                                                                                                                                                                   | `0.16.0`      |
-| `acs-report-enable` | Optionally, enable the feature that causes a result.sarif report to be generated after successful action execution. This report is compatible with GitHub Automated Code Scanning (ACS), as the artifact to upload for display as a Code Scanning Alert report.                                                | `false`       |
+| `fail-build`        | Fail the build if a vulnerability is found with a higher severity. That severity defaults to `"medium"` and can be set with `severity-cutoff`.                                                                                                                                                                 | `true`        |
+| `acs-report-enable` | Optionally, disable the feature that causes a result.sarif report to be generated after successful action execution. This report is compatible with GitHub Automated Code Scanning (ACS), as the artifact to upload for display as a Code Scanning Alert report.                                               | `true`        |
```

Comment:
> Optionally, disable the feature that causes a result.sarif report to be generated after successful action execution.

I think this phrasing could be more straightforward (which is mostly a comment on the prior state, not your change). All of these config options that have default values are "optional", and, IMHO, I think it's easier to read boolean statements when they're not negated.

What about something like...

"Generate a `result.sarif` report after successful action execution."

Reply:
I've updated the text here to also to remove the reference to `results.sarif`, as this might change and the end user should use the output parameter `sarif`.